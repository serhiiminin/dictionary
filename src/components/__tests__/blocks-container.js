import React from 'react';
import renderer from 'react-test-renderer';
import { BlocksContainer } from '..';


describe('Blocks Container', () => {
  const component = renderer.create(
    <BlocksContainer>Anything</BlocksContainer>
  );

  test('render', () => {
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
