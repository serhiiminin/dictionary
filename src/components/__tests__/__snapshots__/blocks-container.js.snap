// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blocks Container render 1`] = `
<div
  className="BlocksContainer-root-0-1-1"
>
  Anything
</div>
`;
